tannerhodges/twitter-print-styles
# 🐦 Twitter Print Styles v2

962 days later…

![Example of Twitter Print Styles v2](https://user-images.githubusercontent.com/1619485/91786205-128ccc00-ebd5-11ea-959d-507ad8343f4d.png)

Obviously room for improvement, but hopefully this update gets the extension working well enough for everyone who's been patiently waiting to use it.

Sorry it took so long!

## How to Test

If you want to test the changes while we wait on review, you can check out https://github.com/tannerhodges/chrome-twitter-print-styles/pull/5 and follow these steps:

1. Download the ZIP: [chrome-twitter-print-styles-v2.zip](https://github.com/tannerhodges/chrome-twitter-print-styles/files/5153538/chrome-twitter-print-styles-v2.zip)
2. Unzip the folder.
3. Go to `chrome://extensions/`.
4. Click **Load unpacked** and select the unzipped folder.
5. Confirm the extension shows up under **Extensions** in the top right of your browser.
6. Go to twitter.com and test it on a few tweets.

👇 If you run into any issues, comment below to let me know.

## Changed

- Rewrite JS & CSS to work with the latest version of Twitter.
- Allow extension to run on all Twitter pages (since Twitter is a SPA now).
- Update JavaScript code to ES6.
- Update `package.json` (and remove yarn).
- Update linters and formatters (stylelint, eslint, editorconfig).

## Fixed

- Fixes #2.
